### ISSUE TITLE
The datasets are unreconstructable.

### ISSUE THREAD

**Issue by TobeyYang** [golsun/StyleFusion#4]
First, thanks for your inspiring research. I am also major in the dialogue system and very interested in this work.
I ran the given scripts in `StyleFusion` to reconstruct the Reddit datasets and found some steps were missing and only some `.tsv` files are obtained in the end.  It is hard to follow now. Could you please provide the `train.num`, `vali.num` and `test.num` and `vocab.txt` in your experiments instead of toy data, thank you.

**Comment by mattc95** [comment]
I have encountered the same issue above.
